firebase-abt/src/main/java/com/google/firebase/abt/component/package-info.java

```diff
@@ -0,0 +1,4 @@
+/**
+ * @hide
```

Comment:
I think the whole comment needs to be on one line, let's see if google-java-format fails on this.

Reply:
PTAL

Reply:
there is a merge conflict in both files, pls double check

Reply:
fixed